src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -42,93 +38,36 @@ <h3 class="slds-section__title slds-theme_shade">
-                                </lightning-layout-item>
-
-                                <lightning-layout-item padding="horizontal-small" size="12"  small-device-size="6">
-                                    <template if:true={isMultiCurrencyEnabled}>
-                                        <div class="slds-p-horizontal_xx-small slds-m-bottom_x-small">
-                                            <label for="currencyField" class="slds-form-element__label slds-no-flex">{fields.currency.label}</label>
-                                            <lightning-input-field id="currencyField" field-name={fields.currency.apiName} variant="label-hidden" requried>
-                                            </lightning-input-field>
-                                        </div>
-                                    </template>
-                                </lightning-layout-item>
-
-                                <lightning-layout-item padding="horizontal-small" size="12"  large-device-size="3" small-device-size="6">
-                                    <lightning-input-field field-name={fields.installmentFrequency.apiName} required value={defaultInstallmentFrequency}>
-                                </lightning-input-field>
-
-                                </lightning-layout-item>
-                                <lightning-layout-item padding="horizontal-small" size="12"  large-device-size="3" small-device-size="6">
-                                    <lightning-input-field field-name={fields.period.apiName} required>
-                                    </lightning-input-field>
-                                </lightning-layout-item>
-
-                                <lightning-layout-item padding="horizontal-small" size="12"  large-device-size="3" small-device-size="6">
-                                    <lightning-input-field field-name={fields.dayOfMonth.apiName} value={defaultDayOfMonth}>
-                                    </lightning-input-field>
-                                </lightning-layout-item>
-
-                                <lightning-layout-item padding="horizontal-small" size="12"  large-device-size="3" small-device-size="6">
-                                    <lightning-input-field field-name={fields.startDate.apiName} required>
-                                    </lightning-input-field>
-                                </lightning-layout-item>
-
-                                <lightning-layout-item padding="horizontal-small" size="12"  small-device-size="6">
-                                    <lightning-input-field field-name={fields.paymentMethod.apiName}>
-                                    </lightning-input-field>
-                                </lightning-layout-item>
-                            </lightning-layout>
-                        </div>
-                    </div>
+                    <c-rd2-entry-form-schedule-section record-id={recordId}></c-rd2-entry-form-schedule-section>
```

Comment:
New Schedule section (Donation Information) is self contained and will load fields data. The only param required is the `recordId`. Works for both new and existing RD.